project/JavaFormatter.scala

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2015 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.google.googlejavaformat.java.{Formatter, JavaFormatterOptions}
+import sbt.Keys._
+import sbt._
+import sbt.util.CacheImplicits._
+import sbt.util.{CacheStoreFactory, FileInfo, Logger}
+import scala.collection.immutable.Seq
+
+object JavaFormatter {
```

Comment:
That file probably needs a similar note about c&p

Reply:
Added in https://github.com/enso-org/enso/pull/8543/commits/fdf6d5da5b19adb0b650527bfbea1c66c4d10562